### DISCUSSION TITLE
Should medium change approval include dev community

### DISCUSSION THREAD

**Discussion by tijno** [deso-protocol/dips#12] [Large CIPs]
Right now medium level changes can only be approved by core devs.

This should include community devs too, but at a lower level than large changes.

Proposed change:

medium: ~~1 core dev and 1 community dev~~ no change needed, see update below
large: 2 core dev and 3 community dev (see update below)

**Comment by F041** [comment]
I think that since we have VCs, we can't have a majority of community devs. So no more decentralization.

**Reply by tijno** [reply]
I dont think VCs change this.

VCs have bought in just like you and i (although earlier and much larger). We are all VCs in a way.

I think "majority" suggests we are on different sides. Which is something we would want to move away from.

For me its more a evolution.

At the start it was just core devs
And there was no community

In future there wont be any distinction between the original core team and community. 

Theres just "us".

**Comment by HPaulson** [comment]
I think it needs to be more than just 1 or 2 community members. I mentioned this privately to Maebeam earlier today, but I think 2:3 core:community ratio is viable at current scale.

**Reply by tijno** [reply]
which level change - the large one ?

im not sure a majority system is right for large changes at this point in time.

cips are untested, and community that can approve is undefined.

just being cheeky here:

giving community majority would allow us to push through a change that gives all of unreserved coins to you me and paul 😉

**Reply by HPaulson** [reply]
To clarify: My suggestion would require ALL 5, 2 core and 3 community, approval for a change to exist. Though, it could be any 2 core and 3 community. It's not a majority as it wouldn't be either or, it'd be both.

**Reply by tijno** [reply]
Ah ok understood @HPaulson - so any of the 5 can stop a change going through, but no-one can force a change without approval from all.

Still though - for which level?

5 approvals for a medium level change seems like overkill? 

I am in favour of not making CIPs too beaurocratic until we are at a scale where its needed.

**Reply by HPaulson** [reply]
@Tijn Five for large. As far as I'm concerned, Small & Medium changes aren't even a big deal as they're related to the core team's frontend and their backend api. These codebases are their implementation built on top of the chain, especially the frontend, and as such they can do as they choose in that regard. Them relying on CIPs is just an extra gift of sorts.

The protocol, however, isn't theirs -- as such, large changes most definitely should require more community reviews.

**Reply by HPaulson** [reply]
Another mention here -- it could be any combination of 2 core devs and 3 community reviews, making it still challenging to "stop" a CIP unless there's huge consensus that it's a bad idea, such as Core #26, where 0 community members (at least from what I saw) agreed with the change.

**Comment by tijno** [comment]
Just checking in on this - 

conclusion from discussions above seems:

1 - medium: as it involves largely frontend changes, discussion as CIP + core team approval is sufficient - so no change needed.

2 - large: adjusted proposal by @HPaulson 2 core + 3 community - 5 approvals in total - required before change can be implemented. 

So suggested change to large approval process:

from 

```
3. The community and core devs will review your CIP PR and provide feedback. 
Once at least two core devs approve, your PR can be merged and is now an official CIP.
```

to

```
3. The community and core devs will review your CIP PR and provide feedback. 
Once at least two core devs & 3 community devs approve, 
your PR can be merged and is now an official CIP. 
```
